Examples/Algorithms/Geant4/src/GdmlDetectorConstruction.cpp

```diff
@@ -1,23 +0,0 @@
-// This file is part of the Acts project.
-//
-// Copyright (C) 2017-2020 CERN for the benefit of the Acts project
-//
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-#include "ActsExamples/Geant4/GdmlDetectorConstruction.hpp"
-
-#include <G4GDMLParser.hh>
-
-using namespace ActsExamples;
-
-GdmlDetectorConstruction::GdmlDetectorConstruction(std::string path)
-    : G4VUserDetectorConstruction(), m_path(std::move(path)) {}
-
-G4VPhysicalVolume* GdmlDetectorConstruction::Construct() {
-  G4GDMLParser parser;
-  // TODO how to handle errors
-  parser.Read(m_path);
-  return parser.GetWorldVolume();
-}
```

Comment:
So this is replaced/superceded by GDML construction in GeoModelG4?

Reply:
Yes, @FabianKlimpel and I have seen issues loading directly the `gdml` via the `G4GDMLParser`, so we have decided to include `GeoModel` to do it. More general, plugins available in `GeoModel` allow to build geometries from several sourced, e.g. `*.db` or `*.xml`, sharing the same files ATLAS uses.

Reply:
Ok, great. If we don't expect a use case for importing GDML bypassing GeoModel, I think this is fine then.